packages/react-jss/src/types.js

```diff
@@ -22,20 +20,31 @@ export type Context = {|
   disableStylesGeneration: boolean
 |}
 
-export type HOCProps<Theme, Props> = Props & {|
+export type HOCProps<Theme, Props> = Props & {
   theme?: Theme,
   jssContext: Context,
   innerRef: any
-|}
+}
+
+export type Classes = {[string]: string}
 
 export type InnerProps = {
   children?: Node,
-  classes: {}
+  classes: Classes
 }
 
 export type DynamicRules = {
   [key: string]: BaseRule
 }
 
-export type ThemedStyles<Theme> = (theme: Theme) => StaticStyles
+type StaticStyle = {}
```

Comment:
we should use this one from the core too, for the sake of a sigle source of truth